config/common/group_vars/k8s_cluster/ck8s-k8s-cluster.yaml

```diff
@@ -41,7 +41,9 @@ enable_nodelocaldns: false
 
 kubernetes_audit: true
 audit_log_path: "/var/log/audit/kube-apiserver-audit.log"
-audit_log_maxage: 7
+audit_log_maxage: 30
```

Comment:
Is this only to make kube-bench happy?
For us there is no reason to store audit logs on the nodes for a long period of time as they are shipped to opensearch.

Reply:
I agree to this

Reply:
Yeah it was just to make kube-bench happy. That is true. We can skip this is we want to then.